user_docs/en/changes.t2t

```diff
@@ -22,21 +22,21 @@ What's New in NVDA
 
 
 == Changes ==
-- The list of available languages in the General Settings dialog is now sorted based on language names instead of ISO 639 codes. (#7284)
-- Add default gestures for alt shift tab and windows tab with all supported Freedom Scientific braille displays. (#7387)
+- The list of available languages in NVDA's General Settings is now sorted based on language names instead of ISO 639 codes. (#7284)
+- Added default gestures for Alt+Shift+Tab and Windows+Tab with all supported Freedom Scientific braille displays. (#7387)
 - For ALVA BC680 and protocol converter displays, it is now possible to assign different functions to the left and right smart pad, thumb and etouch keys. (#8230)
 - For ALVA BC6 displays, the key combination sp2+sp3 will now announce the current date and time, whereas sp1+sp2 emulates the Windows key. (#8230)
-- The user is now asked once on NVDA start-up if they are happy sending usage statistics to NV Access when automatically checking for updates. (#8217)
+- After installing 2018.3, the user is asked once when NVDA starts if they are happy sending usage statistics to NV Access when automatically checking for updates. (#8217)

```

Comment:
I don't think this change makes things clearer. It might suggest that the user stats dialog is shown after installing the release of 2018.3, while it might already have been shown after the installation of one of its alpha or beta predecessors.